schemas/test/en/test_list_collector_repeating_blocks_section_summary.json

```diff
@@ -0,0 +1,361 @@
+                                                        "source": "answers",
+                                                        "identifier": "company-or-branch-name"
+                                                    }
+                                                },
+                                                {
+                                                    "placeholder": "date_difference",
+                                                    "transforms": [
+                                                        {
+                                                            "transform": "calculate_date_difference",
+                                                            "arguments": {
+                                                                "first_date": {
+                                                                    "source": "answers",
+                                                                    "identifier": "registration-date"
+                                                                },
+                                                                "second_date": {
+                                                                    "value": "now"
+                                                                }
+                                                            }
+                                                        }
+                                                    ]
+                                                }
+                                            ]
+                                        },
+                                        "answers": [
+                                            {
+                                                "type": "Radio",
+                                                "label": "Has this company been trading in the UK? (Mandatory)",
+                                                "id": "authorised-trader-uk-radio",
+                                                "mandatory": true,
+                                                "options": [
+                                                    {
+                                                        "label": "Yes",
+                                                        "value": "Yes"
+                                                    },
+                                                    {
+                                                        "label": "No",
+                                                        "value": "No"
+                                                    }
+                                                ]
+                                            },
+                                            {
+                                                "type": "Radio",
+                                                "label": "Has this company been trading in the EU? (Mandatory)",
```

Comment:
You've written (Mandatory) but specified `mandatory: false`

Reply:
Woops

Reply:
Done 👍